sqlserver/tests/compose-hc/setup.sh

```diff
@@ -0,0 +1,23 @@
+# Run the setup script to create the DB.
+# Do this in a loop because the timing for when the SQL instance is ready is indeterminate.
+for i in {1..120};
+do
+    /opt/mssql-tools/bin/sqlcmd -S localhost -U sa -P $SA_PASSWORD -d master -i is_running.sql -b
+    if [ $? -eq 0 ]
+    then
+        echo "INFO: sqlserver is running."
+        break
+    else
+        echo "INFO: sqlserver not running yet. Retrying in 1 second."
+        sleep 1
+    fi
+done
+
+/opt/mssql-tools/bin/sqlcmd -S localhost -U sa -P $SA_PASSWORD -d master -i setup.sql -b
```

Comment:
That `setup.sql` file is fairly big (`> 1 MB`) and might not be ideal to store in git. Would it make sense to generate the SQL instead of keeping it as a static file in the repository?